packages/tree-extension/src/index.ts

```diff
@@ -93,10 +94,87 @@ const createNew: JupyterFrontEndPlugin<void> = {
   }
 };
 
+function activateNotebookTreeWidget(
```

Comment:
Any specific reason to move the activate function to a separate function?

Maybe we can keep inlined as part of the plugin definition for consistency with the rest of the code base?

Reply:
Not really, I reverted it